django/db/models/fields/__init__.py

```diff
@@ -873,7 +873,7 @@ def value_from_object(self, obj):
         return getattr(obj, self.attname)
 
 
-class AutoField(Field):
+class SmallAutoField(Field):
```

Comment:
This should be kept as-is. We use the `Small` wording for small ints and omit the prefix for 32-bit integers. e.g. `SmallIntegerField`, `IntegerField`, and `BigIntegerField`.